Stack/Opc.Ua.Core/Stack/Tcp/UaSCBinaryChannel.cs

```diff
@@ -577,6 +577,11 @@ protected static ServiceResult ReadErrorMessageBody(BinaryDecoder decoder)
                 reason = new UTF8Encoding().GetString(reasonBytes, 0, reasonLength);
```

Comment:
using the Encoding.UTF8 singleton might be better here than allocating many encoders.

Reply:
good catch, all replaced